var/spack/repos/builtin/packages/ncl/package.py

```diff
@@ -19,9 +19,10 @@ class Ncl(Package):
 
     url = "https://github.com/NCAR/ncl/archive/6.4.0.tar.gz"
 
+    version('6.5.0', '133446f3302eddf237db56bf349e1ebf228240a7320699acc339a3d7ee414591')
     version('6.4.0', 'd891452cda7bb25afad9b6c876c73986')
 
-    patch('spack_ncl.patch', when="@6.4.0")
+    patch('spack_ncl.patch')
     # Make ncl compile with hdf5 1.10
     patch('hdf5.patch', when="@6.4.0")
```

Comment:
what about this patch?

Reply:
The remaining two patches have been applied upstream in 6.5.0.